extensions/notebook/package.json

```diff
@@ -360,6 +365,10 @@
           "command": "books.sqlserver2019",
           "when": "sqlserver2019"
         },
+        {
+          "command": "books.hybridjupyterbook",
+          "when": "hybridjupyterbook"
```

Comment:
Is this necessary? You already have your own command to open the book so it seems like adding another one is unnecessary. Plus I'm not even sure where the hybridjupyterbook is coming from so it seems like this can probably just be removed?